### DISCUSSION TITLE
Submitting photos

### DISCUSSION THREAD

**Discussion by Vemenous-Repentile** [endless-sky/endless-sky#10269] [Q&A] [answered]
Apologies if it's somewhere obvious and I've missed it, but is there an easy place to deliver image files that anyone looking to build plugins or those working on the latest version of ES can use?

Love the lore building in the planet descriptions and the snapshots we get of those places. 

Speaking as someone who lives in suburban London but knows a few good places to take landscape shots of the city from angles where it's not too obvious that it's London, I'd love to contribute!

Also up for proofreading/copywriting for anyone working on plugins etc

**Comment by TheGiraffe3** [comment]
You can just make a pull request with the photos in it, though they should be 1440x720 pixels so that they can be used for both high dpi and be downscaled for standard. And helping review pull requests would be great, you don't only have to do plugins.

**Reply by RisingLeaf** [reply]
remember that there should be 2x for the images as well for the highdpi repo

**Reply by Saugia** [reply]
Yep. The images should actually be taken at 1440 x 720. Then that can be scaled down to 720 x 360 for 1x sizes. (Taking the 1x and scaling that up for 2x = big bad.)

**Comment by mOctave** [comment, marked as answer]
> Apologies if it's somewhere obvious and I've missed it, but is there an easy place to deliver image files that anyone looking to build plugins or those working on the latest version of ES can use?

Not particularly easy, but if you have some experience with Git/GitHub you can open a PR to add them to the game directly (see https://github.com/endless-sky/endless-sky/discussions/7107#discussioncomment-5081217 and https://github.com/endless-sky/endless-sky/pull/8550). Otherwise some people post images on the Discord server (https://discord.gg/ZeuASSx) or just put them in a discussion here. If you're happy for them to be used for plug-ins, I'd be careful to specify that as well as the license you're publishing them under and the name/email you want credited (pseudonyms and auto-generated GitHub emails totally fine).
